main.js

```diff
@@ -0,0 +1,34 @@
+const tweeter = Tweeter()
+const renderer = Renderer()
+
+const render = () => renderer.renderPosts(tweeter.getPosts())
+render()
+
+// On click of Twit button
+const post = () => {
+    tweeter.addPost($("#input").val())
+    render()
+}
+
+// On click of delete post button
+$("#posts").on("click", ".delete", function () {
+    tweeter.removePost($(this).closest(".post").data().id)
```

Comment:
Store the `postId` in a variable and then on the next line pass it through. More readable that way.